heartbeat/heartbeat.full.yml

```diff
@@ -46,6 +46,22 @@ heartbeat.monitors:
   # Waiting duration until another ICMP Echo Request is emitted.
   wait: 1s
 
+  # The tags of the monitors are included in their own field with each
```

Comment:
This must be added to `heartbeat/_meta/beat.full.yml` and then you have to run `make update`.